corehq/apps/sms/static/sms/js/add_gateway.js

```diff
@@ -1,6 +1,8 @@
-hqDefine('sms/js/add_gateway', function() {
-    var initialPageData = hqImport('hqwebapp/js/initial_page_data'),
-        AddGatewayFormHandler = hqImport('sms/js/add_gateway_form_handler').AddGatewayFormHandler;
+hqDefine('sms/js/add_gateway',[
```

Comment:
this needs underscore: https://github.com/dimagi/commcare-hq/blob/2f6739e7b736c813681182e8ee7852c7ce027f3d/corehq/apps/sms/static/sms/js/add_gateway.js#L36

Reply:
oh no. I missed that little `_`. thanks

Reply:
added it